WaterLog_Backend/Controllers/SegmentEventsController.cs

```diff
@@ -82,22 +82,31 @@ public async Task AddSegmentEvent([FromBody] SegmentEventsEntry value)
         //Routing to get all currently opened events
         [Route("GetAlerts")]
         public async Task<List<GetAlerts>> GetAlerts()
+
         {
             try
             {
                 //Get all leaks first
-                var leaks = await _db.SegmentLeaks.Where(a => a.ResolvedStatus == EnumResolveStatus.UNRESOLVED).ToListAsync();
+                var leaks = await _db.SegmentLeaks.ToListAsync();
+                leaks = leaks.OrderByDescending(a => a.OriginalTimeStamp).OrderByDescending(a => a.ResolvedStatus).ToList();
                 if (leaks != null)
                 {
                     List<GetAlerts> alerts = new List<GetAlerts>();
                     var proc = new Procedures(_db, _config);
                     foreach (SegmentLeaksEntry entry in leaks)
                     {
+                        double totalSystemLitres = -1, litresUsed = -1, 
```

Comment:
Declaring doubles but assigning ints. If you make use of a double your initialisation of these should be `-1.0`

Reply:
Done